client/test/backend.go

```diff
@@ -394,6 +397,52 @@ func (b *MockBackend) removeSubscription(ch channel.ID, sub *MockSubscription) {
 	b.eventSubs[ch] = append(subs[:i], subs[i+1:]...)
 }
 
+// funder mocks a funder for the MockBackend.
+type funder struct {
+	rng       rng
+	mtx       sync.Mutex
+	fundedWgs map[channel.ID]*sync.WaitGroup
+}
+
+// newFunder returns a new funder.
+func newFunder(rng rng) *funder {
+	return &funder{
+		rng:       rng,
+		fundedWgs: make(map[channel.ID]*sync.WaitGroup),
+	}
+}
+
+// initFunder initializes the funded WaitGroups for a channel if not already
+// initialized.
+//
+// Must be called before using the Funder for a funding request.
+func (f *funder) initFunder(req channel.FundingReq) {
```

Comment:
```suggestion
func (f *funder) InitFunder(req channel.FundingReq) {
```

Even if the object itself is private, its interface used by other components should be public. Same for other functions that are called from other components (i.e., `fund` and `waitForFunding`).